roles/translator/README.md

```diff
@@ -21,35 +21,33 @@ The most typical high level configuration is:
 
 ## Setup
 ### Configuration File
-The `proxy-config-example.toml` is a configuration example which can be copy/paste into `/conf` directory by the party that is running the Translator Proxy (most
+The `tproxy-config-local-jdc-example.toml` is a configuration example  directory by the party that is running the Translator Proxy (most
```

Comment:
There are two config-examples files, one that needs to be run if you want to use JobDeclarationProtocol (so you run JDC and JDS), the other if you are not using it (your templates are built by the pool).
Have a look at SV2 possible configurations in the homepage here: stratumprotocol.org